### DISCUSSION TITLE
todays afternoon session we will organize one party

### DISCUSSION THREAD

**Discussion by atomrays** [Atomfire/laptop-prediction#4] [Announcements]
@jadhavmohini1510 bring cake for all of us

**Comment by atomrays** [comment]
> @jadhavmohini1510 bring cake for all of us

sorry i already bought this

**Comment by jadhavmohini1510** [comment]
ya sure but my birthday is on 15th oct

**Reply by atomrays** [reply]
> ya sure but my birthday is on 15th oct

ok
